mdtf_test_data/synthetic/vertical/cmip_vertical_coord.py

```diff
@@ -0,0 +1,70 @@
+""" Module for generating synthetic CMIP datasets """
+
+___all__ = [
+    "cmip_vertical_coord",
+]
+
+import xarray as xr
+import numpy as np
+
+
+def cmip_vertical_coord():
```

Comment:
Looks good!  Can you please add a unit test for this to `tests/test_synthetic_data.py`?